server/s3server.cc

```diff
@@ -531,88 +532,66 @@ void fini_auth_ssl() {
 int create_global_index(struct m0_uint128 &root_index_oid,
                         const uint64_t &u_lo_index_offset) {
   int rc;
-  struct m0_op *ops[1] = {NULL};
-  struct m0_op *sync_op = NULL;
-  struct m0_idx idx;
-  unsigned short motr_op_wait_period =
-      g_option_instance->get_motr_op_wait_period();
 
-  memset(&idx, 0, sizeof(idx));
-  ops[0] = NULL;
+  const auto motr_op_wait_period = g_option_instance->get_motr_op_wait_period();
+  unsigned n_retry = g_option_instance->get_max_retry_count() + 1;
```

Comment:
why +1. 3 times retry is good

Reply:
It's for compensating prefix decrement.
for (; --n_retry > 0; ::sleep(1))